src/plugins/widgets/time/Time.tsx

```diff
@@ -6,10 +6,23 @@ import Analogue from './Analogue';
 import Digital from './Digital';
 import { Props, defaultData } from './types';
 import './Time.sass';
+import { utcToZonedTime } from 'date-fns-tz';
 
 const Time: FC<Props> = ({ data = defaultData }) => {
-  const { hour12, mode, showDate, showMinutes, showSeconds } = data;
-  const time = useTime();
+  const {
+    hour12,
+    mode,
+    name,
+    showDate,
+    showMinutes,
+    showSeconds,
+    timeZone,
+  } = data;
+  let time = useTime();
+
+  if (timeZone) {
+    time = utcToZonedTime(new Date(time), timeZone);
```

Comment:
Is `time` already a Date object?

Reply:
Maybe it should also use the raw/absolute time if it is going to transform it again?

Something like:
```ts
let time = useTime(timeZone ? 'absolute' : 'zoned');
```